src/parser/jobs/rdm/modules/Combos.tsx

```diff
@@ -36,6 +37,22 @@ export class Combos extends CoreCombos {
 	}
 
 	//Overrides
+	override initialise() {
+		super.initialise()
+
+		//Manafication breaks combos, so lets ensure we break when we see it, if we're in a combo!
+		this.addEventHook(
+			filter<Event>()
+				.type('action')
+				.source(this.parser.actor.id)
+				.action(ACTIONS.MANAFICATION.id),
+			this.onTrackedCast)
```

Comment:
_Action:_ Starting a cast of any spell will break either combo for RDM too, so we also need to account for those.  Though starting and cancelling Jolt et al. doesn't waste your Red Mana stack(s) or drop or consume Magicked Swordplay.

Reply:
Will those get caught by Events['damage'] filters?  Since they are damage based spells?  Or do cancelled actions not get picked up?  If they will get caught by Events['damage'] even if they didn't finish casting, then the Core module will handle them as normal; if not then how would one go about taking an action but only if it was cancelled via a filter on Event?  I wouldn't want to hook their action hit if a cast finished and the damage gets caught as we would double evaluate them.

Reply:
The 'prepare' event is the begin cast action and is what does the combo break for damaging spells.  No 'prepare' event is emitted for instants.
The 'action' event is when an action actually resolves and will always come after 'prepare' for channeled spells.
The 'damage' event corresponds to the "Player prepares Skill on Target" line in FFLogs (the calculateddamage) event and is when the damage is being snapshot.

Core combos won't double ding if it gets two recordBrokenCombo events off of the same cast, since the first one from the 'prepare' event will break the combo and clear out the active combo, so when the 'damage' event comes through, it won't see an active combo chain to break.  So we really do want to hook both - if someone breaks by casting a spell with Swiftcast / Acceleration, it'll be an instant and have an 'action' and 'damage' event at the same time but no 'prepare' event, while if someone starts a channel and then interrupts it because they realized they oopsed, there would be a 'prepare' event but no 'damage' event and waiting until whatever they did next will record the wrong combo breaking time (and action if they did something else after the interrupt).